packages/lumx-react/src/components/dropdown/Dropdown.tsx

```diff
@@ -21,7 +18,7 @@ interface DropdownProps extends GenericProps {
     /** Children of the Dropdown. */
     children: React.ReactNode;
     /** Whether a click anywhere out of the Dropdown would close it. */
-    closeOnClick?: boolean;
+    closeOnClickAway?: boolean;
```

Comment:
I think we might have misunderstood this prop from the beginning (even before this PR)
If you look at the angular version, the `closeOnClick` should control if the dropdown should close when clicking **inside** it.
So I guess we could have both props here